crates/milli/src/search/facet/filter.rs

```diff
@@ -1164,7 +1271,11 @@ mod tests {
         index
             .update_settings(|settings| {
                 settings.set_primary_key("id".to_owned());
-                settings.set_filterable_fields(hashset! { S("id"), S("one"), S("two") });
+                settings.set_filterable_fields(vec![
+                    FilterableAttributesRule::Field("id".to_string()),
+                    FilterableAttributesRule::Field("one".to_string()),
+                    FilterableAttributesRule::Field("two".to_string()),
+                ]);
             })
             .unwrap();
 
```

Comment:
Would be nice to have some added tests here, testing "negative rules" (e.g. a first rule that indicates that "toto" is not filterable, then a second rule that would also match "toto" and make it filterable. The expected result is that "toto" is not filterable, as the first matching rule wins) and case with "parent" (e.g. a first rule on "toto.titi", a second one on just "toto". Field "toto" is expected to match the second rule, and "toto.titi" the first one)